doc/version-history.rst

```diff
@@ -2,6 +2,11 @@
 Version History
 ===============
 
+v2.5.2
+------
+
+* Update conda meta.yaml to new specs
```

Comment:
Nitpicking, but I see a very similar version history bullet for v2.5.1. It may help to be a little more specific about what the new specs are. But developers can check the actual changes and the commit messages so no big deal.

Reply:
Put something more descriptive there.